src/components/page-seo/page-seo.tsx

```diff
@@ -0,0 +1,19 @@
+import Head from "next/head";
+import React, { useContext } from "react";
+import { ReactLocaleContext } from "../../contexts/locale-context";
+
+export interface IPageTitleProps {
+	title?: string;
+}
+
+export const PageSEO = ({ title }: IPageTitleProps): JSX.Element => {
+	const { siteTitle, siteSubtitle, pageName } = useContext(ReactLocaleContext);
+	const pageTitle = `${siteTitle} - ${siteSubtitle} - ${title || pageName}`;
+	return (
```

Comment:
I'd use something like

```javascript
[siteTitle, siteSubtitle, (title | pageName)].filter(Boolean).join(" - ")
```
To handle missing/empty data

Reply:
Added snippet.